File: server/src/main/com/broll/gainea/server/core/goals/impl/e3/G_Gomix.java
package com.broll.gainea.server.core.goals.impl.e3;

import com.broll.gainea.server.core.goals.GoalDifficulty;
import com.broll.gainea.server.core.goals.OccupyGoal;
import com.broll.gainea.server.core.map.impl.BoglandMap;

public class G_Gomix extends OccupyGoal {
    public G_Gomix() {
        super(GoalDifficulty.MEDIUM, "Erobere den Kontinent Gomix");
    }

    @Override
    protected void initOccupations() {
        occupy(BoglandMap.Continents.GOMIX);
    }
}

File: server/src/main/com/broll/gainea/server/core/goals/impl/e3/G_Gomix.kt
package com.broll.gainea.server.core.goals.impl.e3

import com.broll.gainea.server.core.goals.GoalDifficulty
import com.broll.gainea.server.core.goals.OccupyGoal
import com.broll.gainea.server.core.map.impl.BoglandMap

class G_Gomix : OccupyGoal(GoalDifficulty.MEDIUM, "Erobere den Kontinent Gomix") {
    override fun initOccupations() {
        occupy(BoglandMap.Continents.GOMIX)
    }
}
